back/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.21 AS builder
+
+WORKDIR /build
+
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+
+COPY . .
+
+ENV CGO_ENABLED=0 GOOS=linux GOARCH=amd64
```

Comment:
Would it be better to make it in separate file

Reply:
There are only three, not sensitive vars, therefore it is simpler for now to declare them in the Dockerfile, as passing them through a file would add unnecessary complications to the building process. It can be reevaluated if a lot more of env vars are deemed necessary in the future.

Reply:
Ok I agree